src/psycopt2d/train_model.py

```diff
@@ -200,11 +206,6 @@ def main(cfg):
 
     run.finish()
 
-    return roc_auc_score(
```

Comment:
```suggestion
    return roc_auc_score(
        eval_dataset[OUTCOME_COL_NAME],
        eval_dataset[y_hat_prob_col_name],
    )

```

We need to return a metric for Optuna :-)